administrator/components/com_modules/tmpl/modules/default.php

```diff
@@ -91,6 +132,13 @@
 					$canEdit    = $user->authorise('core.edit',		  'com_modules.module.' . $item->id);
 					$canCheckin = $user->authorise('core.manage',     'com_checkin') || $item->checked_out == $user->get('id')|| is_null($item->checked_out);
 					$canChange  = $user->authorise('core.edit.state', 'com_modules.module.' . $item->id) && $canCheckin;
+					if ($workflow_enabled)
+					{
+						$transitions = ContentHelper::filterTransitions($this->transitions, (int) $item->stage_id, (int) $item->workflow_id);
+
+						$transition_ids = ArrayHelper::getColumn($transitions, 'value');
+						$transition_ids = ArrayHelper::toInteger($transition_ids);
+					}
```

Comment:
In newer code we use the camelCaps for variables. $transitionIds  instead of $transition_ids for example. It is not always possible (inherited stuff), but where possible but you could keep it in your mind.

Reply:
Thank you, I will keep this in mind